src/main/java/net/dries007/tfc/objects/inventory/fluid/FluidTankCallback.java

```diff
@@ -0,0 +1,22 @@
+package net.dries007.tfc.objects.inventory.fluid;
```

Comment:
Same as above, this would make more sense in the `objects.fluids.capability` package.